personalsite/urls.py

```diff
@@ -34,6 +35,7 @@
     url(r'^blog/add/?$', AddView.as_view(), name='blog_add_view'),
     url(r'^blog/(?P<entry_id>[0-9]+)/?$', EntryView.as_view(), name='entry_view'),
     url(r'^blog/(?P<entry_id>[0-9]+)/delete/?$', EntryDeleteView.as_view(), name='delete_entry_view'),
+    url(r'^blog/(?P<entry_id>[0-9]+)/update/?$', UpdateEntryView.as_view(), name='update_entry_view'),
```

Comment:
i typically call "edit" urls "edit" instead of "update", but that's just personal preference.  You can call it whatever you like.